docs/input_output.md

```diff
@@ -0,0 +1,196 @@
+# File handling with dsub
+
+dsub is designed to make it easy to get input files from a remote storage
+location (such as [Google Cloud Storage](https://cloud.google.com/storage/))
+to your Docker container and then get output files from the Docker container
+back out to a permanent remote storage location.
+
+Rather than giving many options of what disks to allocate and where to
+put input and output files, `dsub` is prescriptive.
+
+* All input and output is written to a single data disk mounted at `/mnt/data`.
+* All input and output paths mirror the remote storage location.
```

Comment:
... mirror the remote storage location with a local path of the form `/mnt/data/gs/bucket/path`.